src/components/users-list.tsx

```diff
@@ -2,18 +2,23 @@ import { FlatList } from "react-native";
 import { Item } from "./user-item";
 import { User } from "../hooks/use-get-users";
 
-interface Props {
+interface UsersListProps {
   users: User[];
+  goToUserPage: (id: string) => void;
```

Comment:
```suggestion
  onUserTap: (id: string) => void;
```

Assim seguimos a convenção de nomenclatura do React.